Sources/EmbraceIO/Options/EmbraceIO+Options.swift

```diff
@@ -2,13 +2,131 @@
 //  Copyright © 2025 Embrace Mobile, Inc. All rights reserved.
 //
 
+import Foundation
+
+#if !EMBRACE_COCOAPOD_BUILDING_SDK
+    import EmbraceCore
+    import EmbraceCommonInternal
+    import EmbraceConfiguration
+    import EmbraceCrash
+    import EmbraceKSCrashBacktraceSupport
+#endif
+
 extension EmbraceIO {
+
+    /// Class used to setup the Embrace SDK.
     public final class Options {
-        // TODO: Build options
-        let appId: String
+        public let appId: String?
+        public let platform: Platform
+        public let endpoints: Embrace.Endpoints?
+        public let captureServices: EmbraceIO.CaptureServicesOptions
+        public let crashReporter: CrashReporter?
+        public let logLevel: LogLevel
+        public let export: OpenTelemetryExport?
+        public let runtimeConfiguration: EmbraceConfigurable?
+        public let processors: [OpenTelemetryProcessor]?
 
-        public init(appId: String) {
+        /// Default initializer for `Embrace.Options` that requires an array of `CaptureServices` to be passed.
+        ///
+        /// If you wish to use the default `CaptureServices`, please refer to the `Embrace.Options`
+        /// initializer found in the `EmbraceIO` target.
+        ///
+        /// - Parameters:
+        ///   - appId: The `appId` of the project.
+        ///   - platform: `Platform` in which the app will run. Defaults to `.iOS`.
+        ///   - endpoints: `Embrace.Endpoints` to be used. Defaults to the normal Embrace based endpoints for the given `appId`.
+        ///   - captureServices: `EmbraceIO.CaptureServicesOptions` instance to configure th
+        ///   - crashReporter: The `CrashReporter` to be installed.
+        ///   - logLevel: The `LogLevel` to use for console logs.
+        ///   - export: `OpenTelemetryExport` object to export telemetry using OpenTelemetry protocols
+        ///   - processors: `OpenTelemetryProcessor` objects to do extra processing
+        public init(
+            appId: String,
+            platform: Platform = .default,
+            endpoints: Embrace.Endpoints? = nil,
+            captureServices: EmbraceIO.CaptureServicesOptions = .init(),
```

Comment:
Not particularly for this PR, but we should decide on using `.init()` or `Name()`. I prefer the latter but understand why sometimes the former is chosen.